docs/features/asc1/stateful/hello_world.md

```diff
@@ -68,10 +69,10 @@ TEAL provides OpCodes allowing the program to _get_ (read) and _put_ (write) dat
 Programs may implicitly read their own _global_ storage and the _local_ storage of the account submitting the _application call transaction_. 
 
 #### get_external
-Reading from _global_ and _local_ storage of an _external_ program or account is allowed by explicitly passing the address as an argument within the _application call transaction_. Programs may read from _global_ storage of up to two (2) _external_ programs. Additionally, programs may read from _local_ storage of up to four (4) _external_ accounts.
+Reading from _global_ and _local_ storage of an _external_ program or account is allowed by explicitly passing the address as an argument within the _application call transaction_. Programs may read from _global_ storage of _external_ programs which are stored in the application array for the specific transaction. Additionally, programs may read from _local_ storage of up to four (4) _external_ accounts which are stored in the accounts array for the specific transaction. For more information on the applications and accounts arrays, see the [stateful smart contract](./index.md#stateful-contract-arrays) documentation.
 
 #### put
-Writing data is restricted to _global_ storage of the "called" program and the _local_ storage of the "calling" account, both specified within the _application call transaction_ (note: _external_ locations may only be read from).
+Writing data is restricted to _global_ storage of the "called" program and the _local_ storage of the "calling" account or any account in the accounts array, both specified within the _application call transaction_ (note: _external_ locations may only be read from). The _local_ storage can only be modified if the account has opted into the application.
```

Comment:
"Writing data is restricted to _global_ storage of the "called" program and the _local_ storage of the "calling" account" --> "Writing data is restricted to _global_ storage of the "called" program and _local_ storage of the "calling" account"